mat64/cholesky.go

```diff
@@ -86,6 +86,21 @@ func (c *Cholesky) Reset() {
 	c.cond = math.Inf(1)
 }
 
+// SetFromUpper sets the Cholesky decomposition from the given
```

Comment:
s/SetFromUpper/SetFromU/ or the converse.

Also, from the given ...? and presumably there should be a warning about the triangle kind and panicking and note that t is not altered or retained.